coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/HiveToTrinoConverterTest.java

```diff
@@ -441,4 +441,16 @@ public void testNegationOperator() {
     String expandedSql = relToTrinoConverter.convert(relNode);
     assertEquals(expandedSql, targetSql);
   }
+
+  @Test
+  public void testAliasOrderBy() {
```

Comment:
Could you please also add the tests for coral-spark?

Reply:
Added in revision